src/config/dynamic-theme-fixes.config

```diff
@@ -14767,6 +14767,15 @@ body,
 .searchTitle {
     background: none !important;
 }
+.sc-classic .header__navMenu > li > a {
+    border-right-color: #242424 !important;
```

Comment:
At default settings this makes the separator invisible.

Reply:
> At default settings this makes the separator invisible.

what do you mean?

Reply:
> At default settings this makes the separator invisible.

yes, so thats why I added that line, to make the separators visible. Since the normal soundcloud also has separators.

Reply:
What I meant, that with default settings of Dark Reader the separator is already visible: 
![image](https://user-images.githubusercontent.com/25481501/161691542-de049cb2-36a9-49b6-86d4-c5ab78487622.png)